src/snowflake/connector/auth/webbrowser.py

```diff
@@ -414,3 +418,13 @@ def _get_sso_url(
         sso_url = data["ssoUrl"]
         self._proof_key = data["proofKey"]
         return sso_url
+
+    @staticmethod
+    def _get_console_login_url(conn: SnowflakeConnection, port: int, user: str) -> str:
+        url = (
+            conn._rest.server_url
+            + "/console/login/?"
+            + urlencode({"login_name": user, "client_port": port})
```

Comment:
Is this how to add two query parameters in Python?

Reply:
We have a unit test for this, please look for `REF_CONSOLE_LOGIN_SSO_URL` in the changes.